site/src/components/Footer.tsx

```diff
@@ -42,13 +42,13 @@ const FooterNavigationLinks = FOOTER_NAVIGATION_LINKS.map(({ href, name }) => (
     component="p"
     variant="bpSmallCopy"
     sx={{
-      "&:first-child": {
+      "&:first-of-type": {
```

Comment:
```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".
```

Background: https://infinum.com/handbook/frontend/react/common-ssr-errors

Not sure if this change can have any negative side-effects, so worth a quick visual check.

Reply:
Did a quick check by looking at the footer navigation links, and seems to have worked 👍